app/views/checkout/payment.scala.html

```diff
@@ -30,7 +31,7 @@
             data-number-of-months="@plan.billingPeriod.numberOfMonths"
             data-currency="@currency"
             value="@plan.productRatePlanId.get"
-            @if(plan == defaultPlan){checked}
+            @if(plan == defaultPlan && currency == defaultCurrency){checked}
```

Comment:
This is the actual fix to correctly pre-select the rate plan radio button on the checkout page.